regression/scripts/Vehicles/Concorde.py

```diff
@@ -147,55 +136,73 @@ def vehicle_setup():
     segment.append_airfoil(wing_airfoil)
     wing.Segments.append(segment)      
     
+    # control surfaces -------------------------------------------
```

Comment:
They don't, but in the configurations section, the take-off configuration has them defined  - the angles are set to 0. I removed them to avoid confusion. I also will remove the lines in the configuration section which previously allowed the user to define something that does not exist